src/apps/vpn/telemetry/pings.yaml

```diff
@@ -16,6 +16,15 @@ vpnsession:
     session-based metrics, etc.
   include_client_id: true
   send_if_empty: false
+  reasons: 
+    flush: |
+      Safety measure for when the vpn session does not end in an expected way, sent right before the start ping
+    start: |
+      When the vpn session starts
```

Comment:
Can you be more specific here about what it means for a session to start i.e. "A session starts when a user turns on the VPN". This is documentation for data analyzers, so it's good to be very specific so they don't need to ask / look at the code.